### PR TITLE
6.x

### PR DESCRIPTION
**Pull request by musabgelisgen**
Changes for ES6

### CODE REVIEW

File: jest-common/src/test/java/io/searchbox/indices/script/CreateIndexedScriptTest.java [hunk 1]
```diff
@@ -31,13 +30,6 @@ public void setUp() {
                 "def test_b=\"$test_a\"\n";
     }
```
Comment (ferhatsb) [inline jest-common/src/test/java/io/searchbox/indices/script/CreateIndexedScriptTest.java]: This test should modified to check default script is painless

Comment (ferhatsb) [reply jest-common/src/test/java/io/searchbox/indices/script/CreateIndexedScriptTest.java]: Also we need to add a test that checks given script language value is properly set.

File: jest/src/test/java/io/searchbox/indices/script/GetIndexedScriptIntegrationTest.java [hunk 2]
```diff
@@ -11,13 +11,20 @@
 
 public class GetIndexedScriptIntegrationTest extends AbstractIntegrationTest {
```
Comment (ferhatsb) [inline jest/src/test/java/io/searchbox/indices/script/GetIndexedScriptIntegrationTest.java]: Test class name should reflect the api -  ...StoredScript..

**Comment by basscce** [comment]
will we have support for ingest attachman?

**Pull request merged**